### ISSUE TITLE
[Feature]: Add .npmrc file to suppress some output from npm

### ISSUE THREAD

**Issue by seriouslysean** [cuttle-cards/cuttle#151]
### Feature Summary

Clean up message output in npm

### Detailed Description

https://twitter.com/wesbos/status/1579895071272755200?s=46&t=pYEfeWIxVs98nek7vXcooA

> take the noise out of your npm install by putting a .npmrc file in your project with these settings
> You should still periodically run `npm audit` and `npm fund`, but this cleans it up for those of us who re-install often

```
fund=false
audit=false
legacy-peer-deps=false
loglevel=error
```

For the settings above:
- `fund`, https://docs.npmjs.com/cli/v8/using-npm/config#fund
- `audit`, https://docs.npmjs.com/cli/v8/using-npm/config#audit
- `legacy-peer-deps`, https://docs.npmjs.com/cli/v8/using-npm/config#legacy-peer-deps
    - This will be required when we upgrade npm to 8 as peer deps are strict and will prevent installs for mismatches. We can remove this once our package.json is fully compliant. cc @itsalaidbacklife 
- `loglevel`, https://docs.npmjs.com/cli/v8/using-npm/config#loglevel

Test the output here and compare it to what we see today to make sure it's actually having an impact. Might be worth looking to see if there are other settings that might be useful to have, too.

- https://docs.npmjs.com/cli/v8/using-npm/config
- https://docs.npmjs.com/cli/v8/configuring-npm/npmrc 
- https://stackabuse.com/the-ultimate-guide-to-configuring-npm/
- https://nodesource.com/blog/configuring-your-npmrc-for-an-optimal-node-js-environment/

**Comment by lbalmaceda** [comment]
I've never done this before. Happy to explore the available options

**Comment by seriouslysean** [comment]
@lbalmaceda would love to have you! New to me, too, so we can learn together, haha.

**Comment by lbalmaceda** [comment]
Seems like `audit=false` would just stop submitting the dependencies to npm for a scan. That makes that last message disappear, but how often do you run package install to consider this message noise? Might be worth keeping around.
<img width="434" alt="Screenshot 2022-10-12 at 15 52 53" src="https://user-images.githubusercontent.com/3900123/195365032-f7eb8bbf-8540-4d27-9217-2ee79d85d116.png">


`fund=false` does remove the following message "xxx packages looking for funding" but does not stop the other messages from appearing.
<img width="264" alt="Screenshot 2022-10-12 at 15 52 48" src="https://user-images.githubusercontent.com/3900123/195365125-9985e504-127e-4a41-a0bc-79760b55c17f.png">

<img width="632" alt="Screenshot 2022-10-12 at 15 47 52" src="https://user-images.githubusercontent.com/3900123/195365341-2b912ad7-c70b-4d1d-b0e7-df2559b7541d.png">


According to stack overflow these packages might be abusing the "scripts" feature and running their own custom scripts to put ads before or after the installation of said packages. 

I found a setting that could stop this `ignore-scripts=true` however, even when the output is clean, it breaks running scripts like `npm run e2e:client`.

i.e. With the following options the output is super clear:

```
fund=false
audit=false
ignore-scripts=true
```

<img width="337" alt="image" src="https://user-images.githubusercontent.com/3900123/195366576-152205d9-4548-4ead-8d2b-9fb46b3a96be.png">

But scripts break 

<img width="358" alt="image" src="https://user-images.githubusercontent.com/3900123/195367401-c47bb31f-20e9-45f3-a010-f7dd56571495.png">

Perhaps we can document to run just the install with this flag and keep it out of the `.npmrc` file?

Something like "install deps with `npm ci --ignore-scripts`". I'll make a quick PR now and check your answers later today.

**Comment by lbalmaceda** [comment]
It turns out `loglevel=warn` is pretty much what you were looking for.
![image](https://user-images.githubusercontent.com/3900123/195436162-d922e4ad-5ddc-4e19-b4ec-d1b4be20850d.png)

 If we only make this change, and keep `audit` and `fund` as defaults, the output is still the same.
![image](https://user-images.githubusercontent.com/3900123/195436842-a5304851-4e6d-460f-8341-d16262fb415e.png)

Let me know in the PR if you want me to keep just the `loglevel` flag or all of them.

**Comment by itsalaidbacklife** [comment]
Closed by #153
